tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -219,7 +219,13 @@ describe(__filename, () => {
     expect(root).toHaveClassName('AddonsByAuthorsCard');
   });
 
-  it('should dispatch a fetch action if no add-ons found', () => {
+  // Since it doesn't go through the logic in componentWillReceiveProps
+  // (of comparing changing props values) when it's remounted  - which is
+  // what happens when it doesn't go back to server in some scenarios - we
+  // we want to always make sure to do a fetch on mount to make sure
+  // we have the latest addons list.
+  // REF: https://github.com/mozilla/addons-frontend/issues/4852
```

Comment:
I am not sure I really understand that comment to be honest. Could you make it shorter and focused on the issue we want to solve?

In addition, we often use `See:` to link to an issue.